src/mono/wasm/runtime/jiterpreter-support.ts

```diff
@@ -1972,7 +1981,9 @@ export function jiterpreter_allocate_tables(module: any) {
         totalSize = (tableSize * 2) + (numInterpEntryTables * interpEntryTableSize) + 1,
         wasmTable = getWasmFunctionTable(module);
     let base = wasmTable.length;
+    const beforeGrow = performance.now();
```

Comment:
alternatively you can use `startMeasure` & `endMeasure`

Reply:
Does that work in v8 console?